eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/AssertTest.java

```diff
@@ -0,0 +1,1221 @@
+package org.eclipse.collections.impl.test;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import org.junit.Assert;
+import org.junit.ComparisonFailure;
+import org.junit.Test;
+import org.junit.function.ThrowingRunnable;
+import org.junit.internal.ArrayComparisonFailure;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+
+/**
+ * Based on the test org.junit.tests.assertion.AssertionTest in JUnit 4, replacing all usages of {@link Assert} with {@link Verify}. Ensures that {@code Verify} remains source compatible with {@code Assert}.
+ */
+@SuppressWarnings("StaticMethodReferencedViaSubclass")
```

Comment:
This suppression stops the IDE from warning that these methods should be called through Assert.